docs/api.md

```diff
@@ -201,6 +201,59 @@ Example Request Body:
 Returns:
 `201` if successful
 
+### POST /v1/dex/orders/replace
+
+Description:
+Replace an existing order in the orderbook.
```

Comment:
Add in something about the canceling of the old order and placing of the new one being atomic